app/src/main/kotlin/com/wire/android/ui/common/RowItemTemplate.kt

```diff
@@ -53,6 +55,7 @@ fun RowItemTemplate(
     eventType: EventType? = null,
     onRowItemClicked: () -> Unit,
     onRowItemLongClicked: () -> Unit,
+    endIcon: (@Composable () -> Unit)? = null,
```

Comment:
I think the convention is `leadingIcon` for starting icons and `trailingIcon` for ending icons if you wanna be consistent with jetpack compose naming.

Reply:
Totally agree! it's a better name :D